release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -152,9 +152,145 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha3](https://githu
 
-##### Bug Fixes
+##### Log Package Updates
 
-To view bug fixes, see the [GitHub milestone for Swan Lake Alpha3](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Swan+Lake+Alpha3%22).
+###### Introduce additional log levels and log functions
+
+There are 4 log levels: `DEBUG`, `ERROR`, `INFO`,and `WARN` and their respective functions: `printDebug`, `printError`, `printInfo`, and `printWarn`. To set the global log level, place the entry given below in the `Config.toml` file:
+```
+[log] 
+level = "[LOG_LEVEL]"
+```
+
+Each module can also be assigned its own log level. To assign a log level to a module, provide the following entry in the `Config.toml` file:
+```
+[[log.modules]] 
+name = "[MODULE_NAME]" 
+level = "[LOG_LEVEL]"
+```
+
+#### OS Package Updates
+
+- Removed the `exec` function.
+
+#### Task Package Updates
+
+The module has been revamped by removing the `Scheduler` and `Listener` classes and introducing the following functions to schedule and manage the job either one-time or periodically.
+
+- Configure the scheduler worker pool with the worker count and max waiting time.
+```ballerina
+task:Error? output = task:configureWorkerPool(6, 7000);
+```
+
+- Schedule the Ballerina job at a specified time.
+```ballerina
+class MyJob {
+   *task:Job;
+
+   public function execute() {
+       // logic goes here
+   }
+}
+
+ZoneOffset zoneOffset = {hours: 5, minutes: 30};
+Civil time = {
+    year: 2021,
+    month: 4,
+    day: 12,
+    hour: 23,
+    minute: 20,
+    second: 50.52,
+    timeAbbrev: "Asia/Colombo",
+    utcOffset: zoneOffset
+};
+task:Error|task:JobId id = task:scheduleOneTimeJob(new MyJob(), time);
+```
+
+- Schedule the recurring Ballerina job according to the given duration.
+```ballerina
+class MyJob {
+   *task:Job;
+
+   public function execute() {
+       // logic goes here
+   }
+}
+task:Error|task:JobId id = task:scheduleJobRecurByFrequency(new MyJob(), 1);
+```
+
+- Unschedule the particular `job`.
+```ballerina
+task:Error? result = unscheduleJob(id);
+```
+
+- Pauses all the jobs.
+```ballerina
+task:Error? result = pauseAllJobs();
+```
+
+- Resumes all the jobs.
+```ballerina
+task:Error? result = resumeAllJobs();
+```
+
+- Pauses the particular job.
+```ballerina
+task:Error? result = pauseJob(id);
+```
+
+- Resumes the particular job.
+```ballerina
+task:Error? result = resumeJob(id);
+```
+
+- Gets all the running jobs.
+```ballerina
+task:Error? result = getRunningJobs();
+```
+
+#### Time Package Updates
+
+Revamped the entire time package as follows:
+
+- Introduced `time:Utc` to represent the UTC timestamp.
+- Introduced `time:Civil` to represent the localized time.
+- Added necessary APIs to do time generation, manipulations, and conversions.
+
+Necessary migrations steps from the previous version to the current version are listed [here](https://github.com/ballerina-platform/ballerina-standard-library/issues/1079).
```

Comment:
```suggestion
Steps for migration from the previous version to the current version are listed [here](https://github.com/ballerina-platform/ballerina-standard-library/issues/1079).
```

IINM, we usually try to avoid links with "here". @praneesha FYA please.

Reply:
Yes. Please add a hyperlink to a text. E.g.,

"Necessary migrations steps from the previous version to the current version are listed [in this issue]".,

Reply:
Updated in 0f32666